src/chainstate/stacks/db/accounts.rs

```diff
@@ -57,6 +57,8 @@ pub struct MinerReward {
     pub tx_fees_streamed_produced: u128,
     pub tx_fees_streamed_confirmed: u128,
     pub vtxindex: u32, // will be 0 for the reward to the miner, and >0 for user burn supports
+    pub from_block_consensus_hash: ConsensusHash,
+    pub from_stacks_block_hash: BlockHeaderHash,
```

Comment:
This isn't the right place for this information.  The `MinerReward` struct is meant only to encode the breakdown of payments to a particular address, for use in the block-processing loop and miner code to know who to credit STX to and how much.  The lack of a block pointer in this struct is intentional -- a particular address may receive different amount of STX at different blocks (for example, Muneeb asked me to consider making it so that transaction fees mature at a different height than the coinbase -- as much as a _month_ later).  Can you put this information somewhere else, like directly into the epoch receipt?